cmd/flux/cmd/compile.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+
+	_ "github.com/influxdata/flux/builtin"
+	"github.com/influxdata/flux/lang"
+	"github.com/spf13/cobra"
+)
+
+// compileCmd represents the compile command
+var compileCmd = &cobra.Command{
+	Use:   "compile",
+	Short: "Compile a Flux script",
```

Comment:
Compile has lots of meanings in the Flux eco system right now, can we be more specific in these descriptions to help clarify which kind of compile we mean. Maybe something like `Compile a Flux script into a query specification.`

Reply:
@nathanielc Was the `.` intended to be included? I just checked a couple of the other commands and there seems to be inconsistent usage of periods.

Reply:
It was not intended. I'd like it to be consistent, but I am not sure we have decided on what that means. I have no opinion either way.